tests/test_benchmark_integration.py

```
@@ -62,7 +62,7 @@ def _load_single_benchmark(name: str) -> dict[str, Any]:
         else:
             result["error"] = "load_tasks returned empty list"
 
-    except Exception as e:  # noqa: BLE001 - intentionally broad for reporting
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)